models/pancreas_ct_dints_segmentation/configs/inference.yaml

```diff
@@ -6,7 +6,7 @@ input_channels: 1
 output_classes: 3
 arch_ckpt_path: "$@bundle_root + '/models/search_code_18590.pt'"
 arch_ckpt: "$torch.load(@arch_ckpt_path, map_location=torch.device('cuda'))"
-bundle_root: "/workspace/MONAI/model-zoo/models/pancreas_ct_dints_segmentation"
+bundle_root: "."
 output_dir: "$@bundle_root + '/eval'"
```

Comment:
cc @dongyang0122 , I changed the bundle_root into ".", this is to align with other bundles as discussed in the ticket: https://github.com/Project-MONAI/model-zoo/issues/247